### PR TITLE
fix: remove exposing to outside

### PR DESCRIPTION
**Pull request by linuxbandit**
Change the expose only to the newly-added healthcheck (#117)

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -20,12 +20,6 @@ services:
     environment:
       NODE_ENV: "${MYAEGEE_ENV}"
       #BUGSNAG_KEY: "${BUGSNAG_KEY_GSUITE_WRAPPER}"
-    labels:
```
Comment (serge1peshcoff) [inline docker/docker-compose.yml:23]: can you instead only expose /healthcheck endpoint? check out mailer on how it's done here

Comment (linuxbandit) [reply docker/docker-compose.yml:23]: ahhh i forgot, yes! now that i merged the other, definitely

**Review by serge1peshcoff — APPROVED** [review]
lgtm

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -23,8 +23,7 @@ services:
     labels:
       - "traefik.backend=gsuite-wrapper"
       - "traefik.port=8084"
-      - "traefik.frontend.rule=PathPrefix:/api/gsuite-wrapper;PathPrefixStrip:/api/gsuite-wrapper"
-      - "traefik.frontend.priority=110"
```
Comment (serge1peshcoff) [inline docker/docker-compose.yml:27]: why removing this though?

Comment (linuxbandit) [reply docker/docker-compose.yml:27]: it's not necessary

**Pull request merged**